core/src/Threads/Kokkos_ThreadsTeam.hpp

```diff
@@ -698,25 +709,63 @@ class TeamPolicyInternal<Kokkos::Threads, Properties...>
         m_team_alloc(0),
         m_team_scratch_size{0, 0},
         m_thread_scratch_size{0, 0},
-        m_chunk_size(0) {
+        m_chunk_size(0),
+        m_tune_team_size(false),
+        m_tune_vector_length(false) {
     init(league_size_request, team_size_request);
     (void)vector_length_request;
   }
 
-  /** \brief  Specify league size, request team size */
+  /** \brief  Specify league size, request team size and vector length*/
```

Comment:
as int the other ones we might just be able to forward consturctors.

Reply:
Deferred to #3412

Reply:
Fixed in 33c6960